tests/assets/move-projects/multiple-signers/sources/Dorm.move

```diff
@@ -0,0 +1,67 @@
+/// Bob's dorm. He offers several hireable flats / apartments for student groups.
+module DeveloperBob::Dorm {
+    use std::signer;
+    use std::vector;
+    use substrate::balance;
+
+    /// Address of the owner of this module.
+    const MODULE_OWNER: address = @DeveloperBob;
+
+    /// Simple solution, fixed price for a flat per month. 
+    const FLAT_PRICE_PM: u128 = 1000000000000; // equals 1 UNIT
+
+    /// Error codes
+    const NOT_MODULE_OWNER: u64 = 0;
+    const MODULE_ALREADY_INITIALIZED: u64 = 1;
+    const NOT_ENOUGH_MONEY: u64 = 2;
+
+    /// Group of students, which rent a flat together.
+    struct Group has store {
+        list: vector<address>
+    }
+
+    /// All tenant groups will be registered here.
+    struct Dorm has key {
+        groups: vector<Group>
+    }
+
+    /// Initialization method for module owner.
+    public fun init_module(module_owner: &signer) {
+        assert!(signer::address_of(module_owner) == MODULE_OWNER, NOT_MODULE_OWNER);
+        assert!(!exists<Dorm>(signer::address_of(module_owner)), MODULE_ALREADY_INITIALIZED);
+        move_to(module_owner, Dorm { groups: vector::empty<Group>() });
+    }
+
+    /// Creates a new tenant group and signs a rental agreement.
+    public fun rent_apartment(
+        student1: &signer,
+        student2: &signer,
+        student3: &signer,
+        months: u8) acquires Dorm
+    {
+        // Check that all of the students have enough money.
+        let per_person = FLAT_PRICE_PM * (months as u128) / 3;
```

Comment:
I'm thinking about the case when amount is not divisible by 3: 
FLAT_PRICE_PM = 1 unit (for easier calculation - I know it expands to much more 😄 ).
months = 10 months

So the flat costs 10 units for 10 months.

Then we'll have:
1 * 10 / 3 = 3

So we'll check further if each student has at least 3 money units. But it will give us 9 (not 10).

Right?

Reply:
Ok, I see that lines 49-51 verify that each user must have more than partial amount (not more or equal).
So still, if address1 has 3, address2 has 3 and address3 has 4 units, (which is 10 in total) they still can't rent a flat as assertion will fail for address1 and address2.

Reply:
The price per month is now divisible by 3 :+1:

Yes, the assumption was, they treat each other fair, so everybody has to pay the same price and needs that amount of tokens :blush: